lib/SILOptimizer/PassManager/PassManager.cpp

```diff
@@ -415,6 +426,13 @@ void SILPassManager::runPassOnFunction(unsigned TransIdx, SILFunction *F) {
   if (SILPrintPassTime) {
     llvm::dbgs() << Delta << " (" << SFT->getID() << "," << F->getName()
                  << ")\n";
+
+    // SWIFT_ENABLE_TENSORFLOW
+    // Write CSV-formatted events, so that we can do aggregate analysis. Format:
+    // [S4TF] Delta,StartTime,PassName,PassType,FuncName
+    // Here PassType is F since it's a function pass.
+    logS4TFPassEvent(StartTime, Delta, SFT->getID(), /*isFunction*/ true,
```

Comment:
nit: Swap StartTime & Delta to match the output?

Reply:
Filed https://bugs.swift.org/browse/SR-9258 to track the upstreaming task.